### DISCUSSION TITLE
Is there a stand-alone "view in AR" button that I can use?

### DISCUSSION THREAD

**Discussion by Michaelfromgo** [google/model-viewer#1877] [Q&A]
### Description
 Is it possible to view my GLTF and USDZ file in AR from clicking a button. I do not want to have a background poster or object preview just a button.

**Comment by Michaelfromgo** [comment]
@cdata @elalish Do you know if its possible to do this?

**Comment by cdata** [comment]
@michaelmudukuti sorry for the delay, we are just getting back from holiday over here.

It is very much possible to do this, but we do not currently have an option to help you do it. We have discussed possibly creating a stand-alone button in #697 . If you would like to try making a button for yourself, I would refer you to the following code that does the majority of the work to launch various other AR viewers: https://github.com/GoogleWebComponents/model-viewer/blob/master/src/features/ar.ts

**Comment by Michaelfromgo** [comment]
Thank you for the response! The link you posted seems to be gone. Is there and updated link? @cdata

**Comment by cdata** [comment]
Yes, sorry, we just moved a bunch of things. The new link is: https://github.com/GoogleWebComponents/model-viewer/blob/master/packages/model-viewer/src/features/ar.ts

**Comment by allenwixted** [comment]
@michaelmudukuti did you have any success with this?

**Comment by leoncvlt** [comment]
As mentioned in #697, I built a quick implementation a component for such a stand-alone button: https://leoncvlt.github.io/ar-button/

**Reply by morandd** [reply]
This is perfect.
(And even more perfect will be when this is combined with model-viewer so we get the USDZ automatically generated.)

**Reply by elalish** [reply]
You can always just `display: none` the MV element and use `activateAR()`.

**Comment by elalish** [comment]
Yes, and we've also made this easier in model-viewer; just set `ar` and `reveal="manual"` and the AR button will show up if available. You can make a custom AR button with the `ar-button` slot and you can size the model-viewer element to match.

**Comment by aahuba** [comment]
@leoncvlt Could you please help me with your button in my case? It doesn`t work for me

**Comment by leoncvlt** [comment]
> @leoncvlt Could you please help me with your button in my case? It doesn`t work for me

Sure, do open an issue over in the repo.
